pandas/core/frame.py

```diff
@@ -728,11 +728,35 @@ def iteritems(self):
         """
         Iterator over (column name, Series) pairs.
 
+        Iterates over columns as key, value dict-like pairs with columns name as keys and Series as values.
+
+        Returns
+        -------
+        it : generator
+            A generator that iterates over the columns of the frame.
+
         See also
         --------
         iterrows : Iterate over DataFrame rows as (index, Series) pairs.
         itertuples : Iterate over DataFrame rows as namedtuples of the values.
```

Comment:
`A` in `See Also` should be capitalized.

Use `DataFrame.iterrows` instead of `iterrows` and same for `itertuples`.